static/test/javascripts/spec/common/identity/Api.spec.js

```diff
@@ -34,8 +34,8 @@ define([
                 };
                                getCookieStub = sinon.stub();
                                getCookieStub.withArgs('GU_U').returns(
-                                   'WyIyMzEwOTU5IiwiamdvcnJpZUBnbWFpbC5jb20iLCJqYW1lc2dvcnJpZSIsIjUzNCIsMTM4Mjk1MzAzMT' +
-                                   'U5MSwxXQ.MC0CFBsFwIEITO91EGONK4puyO2ZgGQcAhUAqRa7PVDCoAjrbnJNYYvMFec4fAY'
+                                   'WyIyMzEwOTU5IiwiamdvcnJpZUBnbWFpbC5jb20iLCJBbcOpbGllIErDtHNlIiwiNTM0IiwxMzgyOTUzMDMxNTkxLDFd' +
```

Comment:
This won't test the JavaScript you've changed.

In fact, we currently have no easy way to test the inlined scripts, excepting Selenium tests (which we probably don't want to use).